src/main.test.js

```diff
@@ -27,3 +27,18 @@ test("Valid input", () => {
       '<input id="env_spec_debug" name="debug" />\n'
   );
 });
+
+test("Wrong value type", () => {
+  const testEnv4 = "DATABASE_URL: notgood\nADMIN_EMAIL\nDEBUG :   text";
```

Comment:
This should result in an error. Even a single invalid line in a `.env.spec` file should render nothing and return `null`.